### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by sigvei** [canton7/SyncTrayzor#120]
This error occured a few weeks ago at first, but upgrading hasn't solved it.

This is the crash:

```
Version: 1.0.25.82; Variant: Installed; Arch: Amd64
Path: C:\Program Files\SyncTrayzor\SyncTrayzor.exe
System.Xml.XmlException: ., heksadesimalverdi 0x00, er et ugyldig tegn. Linje 1, posisjon 1.
   ved System.Xml.XmlTextReaderImpl.Throw(String res, String[] args)
   ved System.Xml.XmlTextReaderImpl.ParseRootLevelWhitespace()
   ved System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   ved System.Xml.Linq.XDocument.Load(XmlReader reader, LoadOptions options)
   ved System.Xml.Linq.XDocument.Load(Stream stream, LoadOptions options)
   ved SyncTrayzor.Services.Config.ConfigurationProvider.LoadFromDisk(Configuration defaultConfiguration) i c:\projects\synctrayzor\src\SyncTrayzor\Services\Config\ConfigurationProvider.cs:linje 142
   ved SyncTrayzor.Services.Config.ConfigurationProvider.Initialize(Configuration defaultConfiguration) i c:\projects\synctrayzor\src\SyncTrayzor\Services\Config\ConfigurationProvider.cs:linje 87
   ved SyncTrayzor.Bootstrapper.Configure() i c:\projects\synctrayzor\src\SyncTrayzor\Bootstrapper.cs:linje 88
   ved Stylet.BootstrapperBase.Start(String[] args)
   ved System.Windows.Application.<.ctor>b__1(Object unused)
   ved System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   ved MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)
```

The Norwegian text translates to: hex value 0x00 is an illegal character. Line 1, position 1.

I tried to start syncthing manually as well, and it fails by saying «FATAL: Cannot open database: leveldb/storage: corrupted or incomplete CURRENT file - Is another copy of Syncthing already running?». Another copy is not running.

I am not sure whether the SyncTrayzor error causes the Syncthing error or vice versa, so I start here. Feel free to close it if it is clearly a Syncthing error, and I will bring it over there for further discussion.

**Comment by sigvei** [comment]
I just fixed the Syncthing error, but the SyncTrayzor error persists.

**Comment by canton7** [comment]
Yeah, they're separate and independent errors. It looks like your disk went foobar and corrupted both Syncthing's database and SyncTrayzor's configuration file?

Can you paste the contents of `C:\Users\You\AppData\Roaming\SyncTrayzor\config.xml`? It looks like something prepended a load of null bytes to it... 

In terms of recovery, renaming  `C:\Users\You\AppData\Roaming\SyncTrayzor\config.xml` to e.g.  `C:\Users\You\AppData\Roaming\SyncTrayzor\config.backup.xml` is a good bet: SyncTrayzor will re-create the configuration next time it starts. You'll lose SyncTrayzor-specific configuration options, but not your Syncthing configuration (e.g. devices and folders).

**Comment by canton7** [comment]
How are you getting on with this?

**Comment by canton7** [comment]
This should be fixed in v1.0.27. Let me know if you still see issues.
